microsetta_interface/routes.yaml

```diff
@@ -586,6 +586,33 @@ paths:
               schema:
                 type: string
 
+  # same as above
+  # TODO: Do we need more response codes appended?
```

Comment:
We can throw 403 for not authorized, and I think 404 if `account_id` is not found